build-managed.sh

```diff
@@ -1,4 +1,4 @@
 #!/usr/bin/env bash
 working_tree_root="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-$working_tree_root/run.sh build-managed $*
```

Comment:
I'm by no means an expert in shell scripting so could you please explain the difference between `$*` and `"$@"`?